docs/zh/integration/offline_data_sources/tidb.md

```diff
@@ -0,0 +1,80 @@
+# TiDB
+
+## 简介
+
+[TiDB](https://docs.pingcap.com/zh/) 是一款开源分布式关系型数据库，支持水平扩缩容、金融级高可用、实时 HTAP、云原生的分布式数据库、兼容 MySQL 5.7 协议和 MySQL 生态等重要特性。OpenMLDB 支持使用 TiDB 作为离线存储引擎，用于读取和导出特征计算的数据。
+
+## 使用
+
+### 安装
+
+[OpenMLDB Spark 发行版](../../tutorial/openmldbspark_distribution.md) v0.8.5 及以上版本使用了TiSpark工具来操作TiDB数据库， 当前版本已包含 TiSpark 3.1.x 依赖（tispark-assembly-3.2_2.12-3.1.5.jarh、mysql-connector-java-8.0.29.jar）。如果TiSpark版本不兼容现有的TiDB版本，你可以从[TiSpark文档](https://docs.pingcap.com/zh/tidb/stable/tispark-overview)查找下载对应的TiSpark依赖，并将其添加到Spark的classpath/jars中。
```

Comment:
0.8.5 can't support tidb?

Reply:
The released version 0.8.5 is not supported. Should it be modified to 0.8.6?

Reply:
> 0.8.5 不支持 TiDB 了？

The released version 0.8.5 is not supported. Should it be modified to 0.8.6?